SHA-1/src/main/java/com/abhijith/hashing/PasswordHashing.java

```diff
@@ -1,28 +1,89 @@
 package com.abhijith.hashing;
 
+import java.io.*;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
+
+
 public class PasswordHashing {
-  private   Map<String, String> map = new HashMap<String, String>();
     private static final String SALT = "abhijith";
+    private Map<String, String> map = new HashMap<String, String>();
 
     public static void main(String args[]) {
```

Comment:
Remove the main method. Write JUnit tests fo authenticate
1. with correct username and password and assertEquals
2. with wrong username and password 
3. correct username and wrong password. etc.,